src/Datasource/InvalidPropertyInterface.php

```diff
@@ -17,6 +17,9 @@
 /**
  * Describes the methods that any class representing a data storage should
  * comply with.
+ *
+ * @method \Cake\Datasource\EntityTrait getInvalid()
```

Comment:
Ahh, I see :)
I mixed that up with this forwarding to classes that implement that method.
Should be `array` then.